modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/fork/ForkDetect.scala

```diff
@@ -0,0 +1,48 @@
+package org.tessellation.sdk.infrastructure.fork
+
+import cats.Monad
+import cats.syntax.flatMap._
+import cats.syntax.functor._
```

Comment:
```suggestion
import cats.syntax.contravariantSemigroupal._
```
replace with this import if you go with `mapN` below

Reply:
Imported `contravariantSemigroupal`.